hive/hive-export-import-metastore.md

```diff
@@ -23,73 +23,98 @@ ms.author: dkakadia
 
 #### Issue:
```

Comment:
Would it make sense to put all the code in a separate repository and give a link? That way it could be easily consumed by customers via git. In fact there is a git repo for the other metastore migration tool. https://github.com/hdinsight/MetastoreMigrationTools Perhaps you could refactor a little and put both in there. Your call

Reply:
I see everything is in the hdiconfigactions blob so I suppose you could call this resolved. Happy to discuss.

Reply:
I have no objection to either.